release/release.go

```diff
@@ -857,5 +893,26 @@ func uploadRelease(v version.Version) {
 			}
 		}
 
+		dls = append(dls, dl)
+	}
+
+	// We only have one version for now, so we can just append one ToolsVersion to the JSON
+	// feed and upload immediately. Supporting more versions will require an additional loop.
+	if v.IsStable() {
+		var feed download.JSONFeed
+		feed.Versions = append(feed.Versions, download.ToolsVersion{Version: v.StringWithoutPre(), Downloads: dls})
+
+		marshalledFeed, err := json.MarshalIndent(feed, "", "  ")
+		check(err, "marshal feed")
+
+		feedFilename := "release.json"
```

Comment:
can we add something to common.yml so that this release json file gets uploaded as an artifact?